test.sh

```diff
@@ -9,6 +9,13 @@ error_handler () {
 
 trap error_handler ERR
 
+echo "go formatting" 1>&2
+test -z "$(find ./meta ./resp ./internal ./tests ./tools -name "*.go" -exec gofmt -l {} \; | tee /dev/stderr)"
+
+echo "go vetting" 1>&2
+go vet ./meta ./resp ./internal/... ./tests ./tools/...
```

Comment:
Ah, I see. `go vet ./...` behaves differently between `1.8` and `1.9`.

Is there any reason we need to keep this project using `1.7` rather than bumping it to `1.9`?